.travis.yml

```diff
@@ -63,8 +63,8 @@ before_script:
 # Default script for release builds.
 script:
 - ./tools/lint.py
-- ./tools/test_format.py
 - ./tools/build.py -C target/release
+- ./tools/test_format.py
```

Comment:
Hi, I may be wrong but there be a ts extension here, not py?

Reply:
No, this script is still in python. (I replaced format.py with format.ts, but test_format.py is still written in python. I think that rewriting that script in deno is another good first issue.)

Reply:
Ah I see 👍